net.sourceforge.symphonic.jsymphonic.yml

```diff
@@ -0,0 +1,37 @@
+id: net.sourceforge.symphonic.jsymphonic
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=host
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+command: jsymphonic
+modules:
+  - name: jsymphonic
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk8/install.sh
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -D ${FLATPAK_ID}.png /app/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -D ${FLATPAK_ID}.desktop /app/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm755 jsymphonic.sh /app/bin/jsymphonic
+      - install -D *.jar /app/jsymphonic/jsymphonic.jar
+    sources:
+      - type: script
+        dest-filename: jsymphonic.sh
+        commands:
+          - mkdir -p /var/config/jsymphonic
+          - cd /var/config/jsymphonic/
```

Comment:
Why is this required? Why can't the app initiate its own config directory?

Reply:
it creates the config in the same directory that it is run, so the config will be different for every place you run it by default. this makes it the same regardless of where it is ran.

Reply:
this is not persistent in any way.

Reply:
Would it be better to leave it as the default behaviour? (Creating the config in the current directory)

Reply:
no. Fixing the code is the best.